examples/Atlas/+atlasParams/Walking.m

```diff
@@ -11,8 +11,8 @@
       obj.whole_body.w_qdd(findPositionIndices(r, 'back')) = .0001;
       obj.whole_body.w_qdd(r.findPositionIndices('back_bkx')) = 0.1;
       obj.whole_body.damping_ratio = 0.5;
-      obj.body_motion(r.findLinkId('pelvis')).Kp = [nan; nan; 20; 20; 20; 20];
-      obj.body_motion(r.findLinkId('pelvis')).damping_ratio = 0.5;
+      obj.body_motion(r.findLinkId('pelvis')).Kp = [nan; nan; 150; 200; 200; 200];
+      obj.body_motion(r.findLinkId('pelvis')).damping_ratio = 0.6;
```

Comment:
These changes were made to match the parameters we use in the DRC sim, which have been working really well, so I think we should keep this change